packages/calcite-components/src/components/navigation-logo/navigation-logo.scss

```diff
@@ -10,27 +23,31 @@
     cursor-pointer
     transition-default
     focus-base
-    no-underline
-    text-0h;
+    no-underline;
   color: inherit;
-  border-block-end: 2px solid transparent;
+  font-size: var(--calcite-font-size-0);
+  line-height: var(--calcite-font-line-height-fixed-lg);
+  border-block-end: var(--calcite-spacing-base) solid var(--calcite-navigation-logo-border-color, transparent);
+  background-color: var(--calcite-navigation-logo-background-color, var(--calcite-color-foreground-1));
```

Comment:
if the background color of the navigation-logo different than the background color of navigation? If not, use the same token for both `--calcite-navigation-background-color`

Reply:
as discussed in teams, navigation-logo is not expected to use outside of navigation. will fallback this to `navigation-background-color`